lang/en.json

```diff
@@ -1,22 +1,42 @@
 {
-  "ACTOR.TypeCustos": "Custos",
-  "ATTRIBUTE.HitPoints": "Hit Points",
-  "ITEM.TypeMeleeWeapon": "Melee Weapon",
-  "LexArcana.Name": "Name",
-  "LexArcana.Attributes": "Attributes",
-  "LexArcana.Biography": "Biography",
-  "LexArcana.HitPoints": "Biography",
-  "LexArcana.Pietas": "Pietas",
-  "LexArcana.Inventory": "Inventory",
-  "LexArcana.Rituals": "Rituals",
-  "LexArcana.Features": "Features",
-  "LexArcana.ItemName": "ItemName",
-  "LexArcana.Encumbrance": "Encumbrance",
-  "LexArcana.Protection": "Protection",
-  "LexArcana.Damage": "Damage",
-  "LexArcana.Range": "Range",
-  "LexArcana.Difficulty": "Difficulty",
-  "LexArcana.Feat": "Feat",
-  "LexArcana.Parry": "Parry"
+  "ACTOR.TypeCustos":             "Custos",
+  "ATTRIBUTE.HitPoints":          "Hit Points",
+  "ITEM.TypeMeleeWeapon":         "Melee Weapon",
+  "LexArcana.PeritiaeName" :      "Peritiae",
+  "LexArcana.Peritia" :
+  {
+      "DeBello":                  "De Bello",
+      "DeNatura":                 "De Natura",
+      "DeSocietate":              "De Societate",
+      "DeMagia":                  "De Magia",
+      "DeScientia":               "De Scientia",
+      "DeCorpore":                "De Corpore"
+  },
+  "LexArcana.Name":               "Name",
+  "LexArcana.NPC":                "NPC",
+  "LexArcana.DefaultRoll":        "Default Roll",
+  "LexArcana.Abilities":          "Abilities",
+  "LexArcana.AddPeritiaeSpecialty": "Add specialty",
+  "LexArcana.AddPeritiaeSpecialtyPromptTitle": "Add specialty to Peritia {peritiaName}",
+  "LexArcana.InputLabelName":     "Name",
+  "LexArcana.InputLabelModifier": "Modifier",
+  "LexArcana.DiceExpressionBalancedPrompt": "Use balanced roll?",
+  "LexArcana.InvalidRoll":        "invalid roll",
+  "LexArcana.ActionAdd":          "Add",
+  "LexArcana.ActionCancel":       "Cancel",
+  "LexArcana.Biography":          "Biography",
+  "LexArcana.HitPoints":          "Hit Points",
+  "LexArcana.Pietas":             "Pietas",
+  "LexArcana.Inventory":          "Inventory",
+  "LexArcana.Rituals":            "Rituals",
+  "LexArcana.Features":           "Features",
+  "LexArcana.ItemName":           "ItemName",
+  "LexArcana.Encumbrance":        "Encumbrance",
+  "LexArcana.Protection":         "Protection",
+  "LexArcana.Damage":             "Damage",
+  "LexArcana.Range":              "Range",
+  "LexArcana.Difficulty":         "Difficulty",
+  "LexArcana.Feat":               "Feat",
+  "LexArcana.Parry":              "Parry"
```

Comment:
Because the Peritiae work quite differently, they are two steps translation. Why exactly I don't remember well, its probably because we started the project with the D&D 5e system files and we took them as an example